tardis/montecarlo/base.py

```diff
@@ -295,8 +295,9 @@ def calculate_radiationfield_properties(self):
         return t_rad * u.K, w
 
     def calculate_luminosity_inner(self, model):
-        return (4 * np.pi * const.sigma_sb.cgs *
+        self.luminosity_inner = (4 * np.pi * const.sigma_sb.cgs *
                 model.r_inner[0] ** 2 * model.t_inner ** 4).to('erg/s')
+        return self.luminosity_inner
```

Comment:
I would prefer if you do the assignment in the run function, i.e. undo this particular change and then set self.luminosity_inner similar to self.time_of_explosion.